packages/dd-trace/src/opentracing/tracer.js

```diff
@@ -5,8 +5,9 @@ const Tracer = opentracing.Tracer
 const Reference = opentracing.Reference
 const Span = require('./span')
 const SpanContext = require('./span_context')
-const Writer = require('../writer')
-const Recorder = require('../recorder')
+const AgentExporter = require('../exporters/agent/exporter')
```

Comment:
Renaming both exporters to `index.js` would allow importing with `../exporters/agent`